mkdocs_static_i18n/plugin.py

```diff
@@ -36,6 +37,10 @@ def on_config(self, config: MkDocsConfig):
         # first execution, setup defaults
         if self.current_language is None:
             self.current_language = self.default_language
+            log.info(
+                f"Building '{self.current_language}' documentation to directory: {config.site_dir}"
```

Comment:
shouldn't we add 

```suggestion
                f"Building '{self.current_language}' documentation to directory: {config.site_dir}/{self.current_language}"
```

Reply:
The `config.site_dir` uses `\` on Windows, also the "Building..." log message for the alternates was in the `on_page_build` event.
Based on your suggestion, I scoped the log message to the `on_config` event, and also add the proper path suffix for each language.